### DISCUSSION TITLE
Renaming x-modelable to x-proxy

### DISCUSSION THREAD

**Discussion by jillesme** [alpinejs/alpine#4335] [3. Feature ideas]
Hello,

I'd like to suggest a major breaking change. When reading the documentation, I was a little confused by `x-modelable`. The docs say:

> x-modelable allows you to expose any Alpine property as the target of the x-model directive.

To me, this sounds exactly like what a proxy would do. It could either be a breaking change, or an addition. This would make more sense to me personally:

```html
  <div x-data="{ target: 1 }">
    <div x-data="{ source: 1 }" x-proxy="source" x-model="target">
      <button x-on:click="source++">Increment</button>
    </div>
    <p x-text="target"></p>
  </div>
```

What do y'all think?

**Comment by ekwoka** [comment]
I think it's meant to indicate how it is linked with the model. I don't think `proxy` is at all clearer. It specifically proxies it to x-model. Making it `modelable`.

It technically entangles the two values, not really proxying them.

**Reply by jillesme** [reply]
Thank you for the clarification. My understanding of x-modelable was wrong. Consider this closed :)
